Upstream-Author: Christos Zoulas <christos@zoulas.com>
Date: Sun Mar 30 21:40:08 2014 +0000
Commit: 0b628762b05e1ba024166067aad704b8e66753cb
Description:
    remove extra line (Christoph Biedl)

--- a/magic/Magdir/palm
+++ b/magic/Magdir/palm
@@ -1,6 +1,6 @@
 
 #------------------------------------------------------------------------------
-# $File: palm,v 1.11 2013/12/31 19:18:02 christos Exp $
+# $File: palm,v 1.12 2014/03/28 19:11:40 christos Exp $
 # palm:	 file(1) magic for PalmOS {.prc,.pdb}: applications, docfiles, and hacks
 #
 # Brian Lalor <blalor@hcirisc.cs.binghamton.edu>
@@ -147,7 +147,6 @@
 # http://web.mit.edu/tytso/www/pilot/prc-format.html
 >0x20		beshort&0xffbe	0
 >>0		string		>\0		Palm OS dynamic library data "%s"
->0		string		>\0		"%s"
 60		string		ptch		Palm OS operating system patch data
 >0		string		>\0		"%s"
 
